ckanext/canada/schemas/presets.yaml

```diff
@@ -3078,6 +3078,8 @@ presets:
     - value: EXE
       mimetype: application/vnd.microsoft.portable-executable
       replaces: [exe]
+    - value: FEFLOW
+      replaces: [feflow]
```

Comment:
replaces is just for formats that were in the previous version of the metadata standard. It doesn't apply to any new types